src/ml-service/packages/ml-api/met-forecast.py

```diff
@@ -0,0 +1,226 @@
+    return static_channels
+    
+def save_weather_forecasts(weather_forecasts):
+    client = bigquery.Client()
+    dataset_ref = client.dataset('thingspeak','airqo-250220')
+    table_ref = dataset_ref.table('met_office_weather_forecast')
+    table = client.get_table(table_ref)
+
+    rows_to_insert = weather_forecasts
+    errors = client.insert_rows(table, rows_to_insert)
+    if errors == []:
+        return 'Records saved successfully.'
+    else:
+        return errors
+    
+  
+def get_location_hourly_weather_forecasts(latitude:float, longitude:float):
+        headers = { 'x-ibm-client-id': MET_API_CLIENT_ID,
+        'x-ibm-client-secret': MET_API_CLIENT_SECRET,
+        'accept': "application/json"
+        }
+
+        parameters = {
+            "excludeParameterMetadata":"true",
+            "includeLocationName":"true",
+            "latitude":latitude,
+            "longitude":longitude
+        }
+
+        api_url= MET_API_URL
+
+        forecast_results = requests.get(api_url, headers=headers, params=parameters)
+
+        return forecast_results.json(), forecast_results.status_code     
+  
+def get_hourly_met_forecastsx(event, context):
+    """
+        gets hourly spot wether forecasts from met 
+        office(https://metoffice.apiconnect.ibmcloud.com/metoffice/production/)
+        saves them to big query table.
+    """
+    pubsub_message = base64.b64decode(event['data']).decode('utf-8')
+    if (pubsub_message == "get_weather_forecasts"):
+        weather_predictions=[]
+        specified_locations = get_all_static_channels()
+        #specified_locationsx = specified_locations[0:2]
+        if specified_locations:
+            for i in range(0, len(specified_locations)):
+                channel_id = specified_locations[i].get('channel_id')
+                latitude = specified_locations[i].get('latitude')
+                longitude = specified_locations[i].get('longitude')
+
+                forecast_results, status_code = get_location_hourly_weather_forecasts(latitude, longitude)
+
+                if status_code == 200 :
+                    features_geometry = forecast_results['features'][0]['geometry']
+                    features_properties = forecast_results['features'][0]['properties']
+                    forecast_timeseries = features_properties['timeSeries']
+                    location = features_properties['location'].get('name')
+                    location_lat = features_geometry.get('coordinates')[1]    
+                    location_long = features_geometry.get('coordinates')[0]
+                    point_elevation = features_geometry.get('coordinates')[2]
+                    distance_from_requested_point = features_properties['requestPointDistance']
+                    created_at  =  datetime.now()
+
+                    if forecast_timeseries:
+                        for i in range(0, len(forecast_timeseries)):
+                            location_name = location    
```

Comment:
This looks a little bit wordy, could we be able to use `destructuring` when assigning these variables?